### ISSUE TITLE
Lack `declare` before class/function

### ISSUE THREAD

**Issue by yfwz100** [wessberg/rollup-plugin-ts#24]
The generated declaration file is listed below:

![图片](https://user-images.githubusercontent.com/983211/60974643-01dc1f80-a35d-11e9-8a45-c865c9505539.png)

**Comment by wessberg** [comment]
Can you provide me with a repro so I can fix this with a test case? I can’t reproduce this easily.

**Comment by yfwz100** [comment]
I'll try to make a sample...

**Comment by elado** [comment]
also experiencing malformed d.ts, with too many `declare`s.

source:

```ts
// m.ts
export const ten = 10;

// index.ts
import * as m from './m';
export { m };
```

generated `index.d.ts`:

```ts
declare const ten = 10;
declare module m {
    declare const ten = 10; // error
}
export { m };
```

error on the inner `declare`:

```
A 'declare' modifier cannot be used in an already ambient context.ts(1038)
```

generated with `tsc --emitDeclarationOnly` output is 2 files (`index.d.ts` and `m.d.ts`):

```ts
// index.d.ts
import * as m from './ten';
export { m };

// m.d.ts
export declare const ten = 10;
```

TypeScript 3.6.3

**Comment by wessberg** [comment]
@yfwz100, @elado, thanks for your comments.
I've fixed both and released it under v1.1.66.
